ml-agents/mlagents/trainers/tests/torch/test_encoders.py

```diff
@@ -82,3 +91,36 @@ def test_visual_encoder(vis_class, image_size):
     sample_input = torch.ones((1, image_size[0], image_size[1], image_size[2]))
     encoding = enc(sample_input)
     assert encoding.shape == (1, num_outputs)
+
+
+@pytest.mark.parametrize(
+    "vis_class_and_size",
```

Comment:
nit: you should be able to do `"vis_class, size"` here, and `test_visual_encoder_trains(vis_class, size)` as the test definition. Not a big deal though.

Reply:
I did not know I could do that. Thanks!